publish-report/README.md

```diff
@@ -0,0 +1,40 @@
+# Publish Playwright report
```

Comment:
As this is kind of a "private" action, would it make sense to put an admonition to the top of the README to "underline" this? E.g.:
```md
> [!IMPORTANT]  
> This action is only for repositories under the `grafana` organisation.
```

Reply:
Yes I think that makes sense, will update the readme. thanks @leventebalogh